presto-hive/src/main/java/io/prestosql/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -122,6 +122,7 @@
     private final DirectoryLister directoryLister;
     private final int loaderConcurrency;
     private final boolean recursiveDirWalkerEnabled;
+    private final boolean ignoreAbsentPartitions;
```

Comment:
try add this consistently in: fields, ctor params, ctor assignments

i know the existing code is a bit messy from this point of view, but eg keep the new filed always right after `recursiveDirWalkerEnabled `